frontend/src/pages/home/presidentielle2022/HomePresidentielle2022.tsx

```diff
@@ -1,110 +1,135 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { Box, Divider, Stack, Button, Typography } from '@mui/material';
 
+import {
+  Box,
+  Divider,
+  Stack,
+  Button,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import Grid2 from '@mui/material/Unstable_Grid2/Grid2';
+
+import { useCurrentPoll, useLoginState } from 'src/hooks';
+import UsageStatsSection from 'src/features/statistics/UsageStatsSection';
 import TitleSection from 'src/pages/home/TitleSection';
 import PollListSection from 'src/pages/home/PollListSection';
-import UsageStatsSection from 'src/features/statistics/UsageStatsSection';
-import AlternatingBackgroundColorSectionList from 'src/pages/home/AlternatingBackgroundColorSectionList';
-import { useCurrentPoll, useLoginState } from 'src/hooks';
 import { PRESIDENTIELLE_2022_SURVEY_URL } from 'src/utils/constants';
 
 const HomePresidentielle2022Page = () => {
+  const theme = useTheme();
   const { t } = useTranslation();
+
   const { isLoggedIn } = useLoginState();
   const { baseUrl, active } = useCurrentPoll();
 
+  const homeSectionSx = {
+    width: '100%',
+    padding: 6,
+    [theme.breakpoints.down('md')]: {
+      padding: 6,
+      px: { xs: 2, md: 6 },
+    },
+  };
```

Comment:
Isn't it equivalent to

```suggestion
  const homeSectionSx = {
    width: '100%',
    padding: 6,
    px: { xs: 2, md: 6 },
  };
```

Reply:
Yes probably, I kept the original code without trying to improve it. I'm going fix this.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1287/commits/6576a8f15fdae9de7c5aba798cd325961a63c301